features/test_feature.py

```diff
@@ -232,7 +232,7 @@ def build_test():
 def run_test():
     if test_config.current_test not in exec_tests:
         return True
-    if test_config.current_test.startswith('text_experimental_obj_') and test_config.device_filter.count("cuda") == 0:
+    if test_config.current_test.startswith(('text_experimental_obj_', 'graphics_interop_')) and test_config.device_filter.count("cuda") == 0:
```

Comment:
if you put the test here, it will run on cuda backend, but in config file you disable the test on cuda backend. So maybe this change is not need.

Reply:
Its actually the opposite, I want to run on the CUDA backend. So, I've updated the config file.